### ISSUE TITLE
Disable charge in certain situations

### ISSUE THREAD

**Issue by noisiver** [liyunfan1223/mod-playerbots#813]
Currently the bots that have some form of charge (warriors and feral druids) like to use it when they shouldn't.

A good example is Ionar in Halls of Lightning. When Ionar starts to disperse the bots start to run away but then they like to charge back in which often results in death.

It would be nice if they could be told not to use charge when he's casting disperse, and the same when Loken uses Lightning Nova.

There are most likely more situations like these but these are the ones that come to mind.

**Comment by kadeshar** [comment]
You can using command.
Example for charge:
 "/w botname ss +11578"
![obraz](https://github.com/user-attachments/assets/4d2d7100-368c-4b6c-a305-e9c49ad5fbf2)
 "/w botname ss -11578"
![obraz](https://github.com/user-attachments/assets/07e4b3ab-ce06-425b-9223-713454b7dba5)

**Comment by noisiver** [comment]
I guess I could do that with macros. The bots have strategies for these encounters so it makes sense that this is handled in those strategies eventually though..

**Comment by kadeshar** [comment]
Yeap is only workaround. Myself i was always somehow lucky and didnt notice that but its happening https://youtu.be/YoD2eJpGvM8?list=PLOjP9sYKFs7xnZWAF_zAEJ6-57gMLNV-e&t=1211

**Comment by noisiver** [comment]
Thankfully it's not just me. I appreciate you telling me about that workaround though, it was very frustrating but I don't have to worry about it for the forseeable future at least. Thank you!

**Comment by Dreathean** [comment]
@noisiver do you think this issue is adequately addressed with the +ss -ss commands? At least the two examples you gave were hardcoded in with https://github.com/liyunfan1223/mod-playerbots/pull/836. I think maybe this is good to close and if there are any other specific encounters perhaps an issue can be raised for those?

**Comment by noisiver** [comment]
Yeah. I haven't tried the change yet but this issue is about those two encounters. If I run into more situations I'll open a new issue.
